internal/sliceutils/sliceutils.go

```diff
@@ -44,6 +44,22 @@ func StringSliceEquals(a, b []string) bool {
 	return true
 }
 
+// StringSliceStartsWith returns true if slice A starts with (in order)
+// the elements of slice B
+func StringSliceStartsWith(a, b []string) bool {
```

Comment:
`a, prefix` would make it clear which one's which without having to read comments/code